examples/inspec2ckl/README.md

```diff
@@ -0,0 +1,43 @@
+# Inspec2CKL Usage information
+
+The InSpec to Checklist Parser scans and extracts the results defined in the Inspec JSON results and converts them into a
+Checklist XML file for use with STIGViewer.
+
+## XCCDF attributes YAML file
+
+### Inclusion of STIG information within the Checklist output.
+
+Inspec is unable to produce certain data that is required for conversion into a fully populated checklist. The following
+attributes supplement this data
```

Comment:
What are your thoughts about supporting generic SIData elements by looping over attributes vs. only supporting title, plaintext, version, date?

Reply:
Hi all, I would like to resolve this. Any thoughts on where we want to go with this PR.

Thanks,

Aaron